package.json

```diff
@@ -51,16 +54,19 @@
     "@enkidevs/assert-restricted-branch": "^1.0.2",
     "@enkidevs/eslint-config-backend": "^0.2.3",
     "babel-cli": "^6.26.0",
+    "babel-eslint": "^8.2.6",
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-stage-0": "^6.24.1",
     "eslint": "^5.8.0",
+    "eslint-config-babel": "^8.0.1",
     "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^8.0.0",
```

Comment:
why do we need to install this and the `eslint-plugin-promise`? They ship with out eslint config

Reply:
eslint throws and errors and recommends to install them otherwise

Reply:
Something is going wrong, maybe a global config, maybe needs fresh `node_modules`

Reply:
`rm -rf`'d em and still same error:
```
Oops! Something went wrong! :(

ESLint: 5.8.0.
ESLint couldn't find the plugin "eslint-plugin-node". This can happen for a couple different reasons:
```

Reply:
Yeah I had this error happen to me before, I forgot how I fixed it 🙃

Reply:
maybe eslint needs updating?